src/main/java/nextstep/subway/path/domain/Path.java

```diff
@@ -0,0 +1,51 @@
+package nextstep.subway.path.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import nextstep.subway.line.domain.Distance;
+import nextstep.subway.station.domain.Station;
+
+public class Path {
+    private static final String CAN_NOT_EMPTY = "경로는 비어있을 수 없습니다.";
+    private final List<Station> stations;
```

Comment:
컨벤션 부분이지만, 클래스 변수와 인스턴스 변수에 구분이 있는게 어떨까요?
습관적으로 가져가봐도 좋을 거 같습니다 :)
```suggestion

    private final List<Station> stations;
```